src/IssueViz.Security/HotspotsControl/HotspotsControl.xaml

```diff
@@ -1,10 +1,8 @@
 ﻿<UserControl x:Class="SonarLint.VisualStudio.IssueVisualization.Security.HotspotsControl.HotspotsControl"
              xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-             xmlns:vsShell="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.14.0"
```

Comment:
We don't need UI themes yet but sadly that's gonna be another problem in the future. We might be able to solve it with extern alias, but I'm not sure that works for xaml. We'll see when we get to it.

Reply:
Presumably you'll hit this problem soon? It would be better to make sure we can work round it sooner rather than later.